config/systemd/omnect-ui.service

```diff
@@ -0,0 +1,42 @@
+[Unit]
+Description=omnect ui
+After=time-sync.target docker.service omnect-device-service.service
+Before=first-boot-complete.target
+Wants=first-boot-complete.target docker.service omnect-device-service.service
+ConditionFirstBoot=yes
+
+[Service]
+Type=oneshot
+
+EnvironmentFile=-/etc/omnect/omnect-ui.env
+
+ExecStart=/bin/bash -c "docker load < /mnt/factory/oci_images/omnect-ui.tar.gz; \
+                        docker run -d --restart always --name omnect-ui \
+                            --user 10000:10000 \
```

Comment:
better use $(id ...) 
if we ever change that.

Reply:
done